### PR TITLE
[BigQuery] Open query editor from view and model details

### PR DESCRIPTION
**Pull request by CynthiaXJia**
Allows user to open a new full-window editor with a starter query from the view and model details pages.

### CODE REVIEW

File: jupyterlab_bigquery/src/components/details_panel/table_details_tabs.tsx [hunk 1]
```diff
@@ -86,7 +86,10 @@ export default class TableDetailsTabs extends React.Component<Props, State> {
                     'main',
                     queryId,
                     undefined,
-                    [queryId, `SELECT * FROM \`${this.props.table_id}\``]
+                    [
+                      queryId,
+                      `SELECT * FROM \`${this.props.table_id}\` LIMIT 1000`,
+                    ]
```
Comment (CynthiaXJia) [inline jupyterlab_bigquery/src/components/details_panel/table_details_tabs.tsx:92]: I changed this to `LIMIT 1000` to mimic the web UI behavior

File: jupyterlab_bigquery/src/components/query_editor/query_text_editor/query_text_editor.tsx [hunk 2]
```diff
@@ -595,7 +596,7 @@ class QueryTextEditor extends React.Component<
     const { iniQuery } = this.props;
 
     // eslint-disable-next-line no-extra-boolean-cast
-    const queryValue = !!iniQuery ? iniQuery : 'SELECT * FROM *';
+    const queryValue = !!iniQuery ? iniQuery : '';
```
Comment (CynthiaXJia) [inline jupyterlab_bigquery/src/components/query_editor/query_text_editor/query_text_editor.tsx:599]: I think an empty editor works nicer than a `SELECT * FROM *` as Polong noted in a feedback comment:

> After installing the extension and clicking on the extension for the first time to see the console in a new tab, the default syntax is:
SELECT * FROM *
>
>It shows an error, which makes me feel like I've done something wrong. It should instead be blank or use a legitimate query

File: jupyterlab_bigquery/src/components/details_panel/table_details_tabs.tsx (hunk 1 above)
Comment (cbwilkes) [reply jupyterlab_bigquery/src/components/details_panel/table_details_tabs.tsx:92]: Thanks this was a friction point that makes this more usable

File: jupyterlab_bigquery/src/components/query_editor/query_text_editor/query_text_editor.tsx (hunk 2 above)
Comment (cbwilkes) [reply jupyterlab_bigquery/src/components/query_editor/query_text_editor/query_text_editor.tsx:599]: An empty query sounds good to me

**Comment by cbwilkes** [comment]
Seems like there are some conflicts to resolve

**Comment by CynthiaXJia** [comment]
> Seems like there are some conflicts to resolve

Should be good to go now!

**Pull request merged**